packages/app/src/app/pages/Dashboard/Components/Header/index.tsx

```diff
@@ -83,7 +83,13 @@ export const Header = ({
               width: 'auto',
             })}
           >
-            + New Folder
+            <Icon
+              name="plus"
+              size={20}
+              title="New"
+              css={css({ paddingRight: 2 })}
```

Comment:
one note on the design system: when we move to the v2 codebase we should aim for limiting these exceptions. the design system should support the spacing in such a way that it is composable via the variants of the components, not with custom alignment